src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -34,8 +34,9 @@ public class Shooter extends SubsystemBase {
 			.radiansPerSecondToRotationsPerMinute(Shooter.gearbox.freeSpeedRadPerSec);
 
 	// PID constants. PID system measures RPM and outputs motor power [-1,1]
-	private static final double P = 0.001500;
+	private static final double P = 0.00170;
 	private static final double I = 0.000100;
+	private static final double leftI = 0;
```

Comment:
This looks like a bad hack. The left and right mechanisms should be identical.

Reply:
We have found that because the left is consistently going above the right by several hundred RPM (because of spin), having no I constant on the left helps it reach setpoint faster and more accurately.

Reply:
I am not sure why my username changed, this is Gavin.